### DISCUSSION TITLE
Endpunkt Impfzertifikatsimulator

### DISCUSSION THREAD

**Discussion by TH-Pega** [Digitaler-Impfnachweis/certification-apis#90] [Q&A] [answered]
Wie lautet der Endpunkt des Impfzertifikatsimulators im Internet (für Variante 1 der Anlage 12.1)?

**Comment by TH-Pega** [comment, marked as answer]
Danke für den Verweis auf #14; werde es also mit https://api-pvs.certify.demo.ubirch.com/api/certify/v2/issue versuchen

**Reply by nicolaslang1** [reply]
Hallo, könnten Sie mir bescheid geben wenn Sie von diesem API Endpunkt das Zertifikat erhalten? Aktuell bekomme ich nur die Meldung, dass mein JWT Token ungültig sei.

**Reply by TH-Pega** [reply]
Ich habe das gleiche Problem wie Sie (#121): Mit client_id "smcb-auth-simulator" wird schon der erste Aufruf abgelehnt, mit client_id "user-access-ti" dann die signierte Challenge. Vermutlich ist die spezielle KoPS-SMC-B nur mit der ersten client_id verknüpft, denn die zweite gilt ja eigentlich für RU/PU. Da die Tokenermittlung bei https://keycloak-cert-auth.dev.ibmega.net problemlos funktioniert, dürfte es sich um einen Fehler bei Ubirch handeln ...
